### PR TITLE
feat: changelog with standard-version

### PR DESCRIPTION
**Pull request by helio-frota**
This is to have auto-generated changelog by using the standard-version.

Related to #14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     "pretest": "eslint --ignore-path .gitignore",
     "lint:fix": "eslint . --fix",
     "test": "jest --no-colors",
-    "coverage": "jest --coverage"
+    "coverage": "jest --coverage",
+		"release": "standard-version"
```
Comment (richardlau) [inline package.json:30]:
```suggestion
    "release": "standard-version"
```

**Comment by helio-frota** [comment]
More information on 
https://github.com/nodeshift/new-module-starter#commit-messages
https://github.com/nodeshift/new-module-starter/pull/7

File: package.json (hunk 1 above)
Comment (helio-frota) [reply package.json:30]: @richardlau thank you for this!
This is not the first time that happens.
https://github.com/nodeshift-starters/reactive-koffeeshop-demo/commit/c68a76eabfce1f65726286bbfa38855fab9e1426

I need to take a look at what is causing it...

Comment (richardlau) [reply package.json]: Probably tabs/spaces (I don't mind which but it should be consistent with the existing file).

**Review by richardlau — APPROVED** [review]

**Review by aalykiot — APPROVED** [review]

**Pull request merged**